web/src/features/map/MapTooltip.tsx

```diff
@@ -56,26 +59,70 @@ function TooltipInner({
   );
   return (
     <div className="w-full text-center">
-      <div className="pl-2">
-        <ZoneName zone={zoneId} textStyle="text-base font-medium" />
-        <div className="flex self-start text-xs">{date}</div>{' '}
-      </div>
-      <div className="flex w-full flex-grow py-1 sm:pr-2">
-        <div className="flex w-full flex-grow flex-row justify-around">
-          <CarbonIntensitySquare intensity={intensity} />
-          <div className="pl-2 pr-6">
-            <CircularGauge
-              name={t('country-panel.lowcarbon')}
-              ratio={fossilFuelPercentage}
-            />
+      <div className="p-3">
+        <div className="flex w-full flex-row justify-between">
+          <div className="max-w-52 pl-2">
+            <ZoneName zone={zoneId} textStyle="font-medium text-base font-poppins" />
+            <div className="flex self-start text-sm text-neutral-600 dark:text-neutral-400">
+              {date}
+            </div>{' '}
+          </div>
+          <DataValidityBadge
+            hasOutage={hasOutage}
+            isEstimated={estimationMethod != undefined}
+          />
+        </div>
+        <div className="flex w-full flex-grow py-1 pt-4 sm:pr-2">
+          <div className="flex w-full flex-grow flex-row justify-around">
+            <CarbonIntensitySquare intensity={intensity} />
+            <div className="pl-2 pr-6">
```

Comment:
I didn't change the padding, as you can see on the deleted line 66, it's the same as before. But yes it might be a good idea to put them in a component!